### PR TITLE
Create new file for places to eat near Carolyn's House

### LINKED ISSUE #10: Carolynshin favorite places to eat
Let's add a document using:
- [x] Create a branch
- [x] Add a new file
- [x] Commit my file
- [x] Open Pull Request
- [x] Discuss
- [x] Merge

### PR DESCRIPTION
**Pull request by carolynshin**
Resolves #10 by adding a file for places to eat near my house!

### CODE REVIEW

**Comment by carolynshin** [comment]
@EmilyUna65, do you have any other recommendations of places to eat in Oakland?!

File: eats/carolyn-neighborhood.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### :sparkles: Carolyn's favorite restaurants in her neighborhood :sparkles:
+
+- [Kamdesh Afghan Kabab House](www.yelp.com/biz/kamdesh-afghan-kabab-house-oakland-2)
+- [Belly](www.yelp.com/biz/belly-oakland-2)
```
Comment (monicanpm) [inline eats/carolyn-neighborhood.md:4]: would you recommend to vegetarians?

Comment (carolynshin) [inline eats/carolyn-neighborhood.md:4]: Unfortunately, Belly is not vegetarian friendly... :cry:

**Comment by cowperthwait** [comment]
lgtm; 🐑

**Comment by watridge** [comment]
:sheep:

**Pull request merged**